src/main/java/com/softserveinc/dokazovi/config/AppProperties.java

```diff
@@ -1,55 +1,71 @@
 package com.softserveinc.dokazovi.config;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 import java.util.ArrayList;
 import java.util.List;
 
 @ConfigurationProperties(prefix = "app")
 public class AppProperties {
-    private final Auth auth = new Auth();
-    private final OAuth2 oauth2 = new OAuth2();
-
-    public static class Auth {
-        private String tokenSecret;
-        private long tokenExpirationMsec;
-
-        public String getTokenSecret() {
-            return tokenSecret;
-        }
-
-        public void setTokenSecret(String tokenSecret) {
-            this.tokenSecret = tokenSecret;
-        }
-
-        public long getTokenExpirationMsec() {
-            return tokenExpirationMsec;
-        }
-
-        public void setTokenExpirationMsec(long tokenExpirationMsec) {
-            this.tokenExpirationMsec = tokenExpirationMsec;
-        }
-    }
-
-    public static final class OAuth2 {
-        private List<String> authorizedRedirectUris = new ArrayList<>();
-
-        public List<String> getAuthorizedRedirectUris() {
-            return authorizedRedirectUris;
-        }
-
-        public OAuth2 authorizedRedirectUris(List<String> authorizedRedirectUris) {
-            this.authorizedRedirectUris = authorizedRedirectUris;
-            return this;
-        }
-    }
-
-    public Auth getAuth() {
-        return auth;
-    }
-
-    public OAuth2 getOauth2() {
-        return oauth2;
-    }
+
+	private final Auth auth = new Auth();
+	private final OAuth2 oauth2 = new OAuth2();
+
+	public static class Auth {
```

Comment:
Please check your IDE settings for indentation and tabs and adjust your code accordingly. Also check other files you have edited. IDE settings should be something like this 
![image](https://user-images.githubusercontent.com/86650723/182669668-061d3ea1-8c76-4571-9508-9cb53c85362c.png)